homeassistant/components/frontend/__init__.py

```diff
@@ -416,27 +454,31 @@ def __init__(self, use_repo):
                 os.path.join(os.path.dirname(__file__), 'templates/')
             )
         )
+        self.js_option = js_option
 
     @asyncio.coroutine
     def get(self, request, extra=None):
         """Serve the index view."""
         hass = request.app['hass']
+        latest = _is_latest(self.js_option, request)
 
         if self.use_repo:
-            core_url = '/home-assistant-polymer/build/core.js'
-            compatibility_url = \
-                '/home-assistant-polymer/build/compatibility.js'
+            core_url = '/home-assistant-polymer/{}/core.js'.format(
+                'build' if latest else 'build-es5')
+            compatibility_url = None
             ui_url = '/home-assistant-polymer/src/home-assistant.html'
             icons_fp = ''
             icons_url = '/static/mdi.html'
         else:
-            import hass_frontend
+            hass_frontend_versioned = _get_frontend_package(latest)
             core_url = '/static/core-{}.js'.format(
-                hass_frontend.FINGERPRINTS['core.js'])
-            compatibility_url = '/static/compatibility-{}.js'.format(
-                hass_frontend.FINGERPRINTS['compatibility.js'])
+                hass_frontend_versioned.FINGERPRINTS['core.js'])
+            compatibility_url = None if latest else \
```

Comment:
Probably more readable if we set `compatibility_url = None` before the if…else

Reply:
Done